pages/custom-query-modules/cpp/cpp-api.md

```diff
@@ -2376,3 +2424,4 @@ During operation, the following exceptions may be thrown.
 | `ImmutableObjectException`    | Object you are trying to change is immutable!   |
 | `ValueConversionException`    | Error in value conversion!                      |
 | `SerializationException`      | Error in serialization!                         |
+| `TextSearchException`         | various (reports text search utility issues)    |
```

Comment:
@antepusic Is this the actual message or do you want to say that in the case of `TextSearchException` various messages can appear?

Reply:
The latter – various error messages can appear